tests/core/pyspec/eth2spec/test/phase0/unittests/fork_choice/test_on_block.py

```diff
@@ -192,21 +95,33 @@ def test_on_block_update_justified_checkpoint_within_safe_slots(spec, state):
 
     next_epoch(spec, state)
     spec.on_tick(store, store.time + state.slot * spec.config.SECONDS_PER_SLOT)
-    state, store, last_signed_block = apply_next_epoch_with_attestations(spec, state, store)
+    state, store, last_signed_block = yield from apply_next_epoch_with_attestations(spec, state, store)
```

Comment:
weird. this works? never seen that before

Reply:
It works!
It seems similar to this explanation: https://www.python.org/dev/peps/pep-0380/#generators-as-threads
We might have overused `yield`. 😅